LocalSettings.php

```diff
@@ -4452,6 +4452,13 @@
 				'rollbacker',
 			),
 		),
+		'+sdiywiki' => array(
```

Comment:
As I said before, this is wrong, only wgGroupPermissions is for defining permissions.

WgAddGroups and wgRemoveGroups is to define what groups can be removed by what groups.